process/OpenHW-Specification-Process.md

```diff
@@ -78,25 +96,25 @@ This section describes the characteristics of the Specification Process. This is
 
 ### Specification Revision Numbers
 
-OpenHW Group Specifications shall use semantic versioning https://semver.org/ with the Revision in the form X.Y.Z.
-The Revision number is combined with the Specification State (below), for example "1.0.0 - Released" to indicate both the Revision number and State.
+OpenHW Group Specifications shall use semantic versioning https://semver.org/ with the Revision in the form vX.Y.Z-postfix
+The Revision number is combined with the Specification State (below), for example "v1.0.0-rel1" to indicate both the Revision number and State.
 
 
 
-### Specification States
+### Specification States and postfix
 
 The ratification or completion State is meant to be written together with the Title and Revision in each OpenHW Group specification so that a reader will be aware of the state of completion of the Specification.
-These are the OpenHW Group Specification States:
+These are the OpenHW Group Specification States and corresponding postfixes:
 
-- “OpenHW Group Specification - In Development"
-- "OpenHW Group Specification - In Review" (optional)
-- "OpenHW Group Specification - Release Candidate"
-- "OpenHW Group Specification - Released"
+- “In Development". The version postfix is -devW, for example -dev1, -dev2 
+- "In Review" (optional). The version postfix is -reviewW, for example -review1, -review2 
```

Comment:
dev and review do not need version numbers beyond the normal version. this is just a special case for release candidates, as they might need to differentiate multiple versions starting with e.g. 1.0.0.
For development versions etc., the main version number would just increment between versions